packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/TwoStepVerification/Yubikey/index.js

```diff
@@ -12,23 +11,29 @@ import Loading from './template.loading'
 class YubikeyContainer extends React.PureComponent {
   constructor (props) {
     super(props)
+    this.state = {
+      updateToggled: false,
+      successToggled: false,
+      yubikeyCode: ''
+    }
     this.handleClick = this.handleClick.bind(this)
     this.onSubmit = this.onSubmit.bind(this)
     this.handleInput = this.handleInput.bind(this)
-
-    this.state = { yubikeyCode: '' }
   }
 
   componentDidUpdate (prevProps) {
     const next = this.props.data.getOrElse({})
     const prev = prevProps.data.getOrElse({})
     if (next.authType !== prev.authType) {
-      this.props.updateUI({ successToggled: true })
+      this.handleUpdate()
       this.props.triggerSuccess()
       this.props.handleGoBack()
       this.props.goBackOnSuccess()
     }
   }
+  handleUpdate () {
```

Comment:
move` setState` line into `componentDidUpdate` and remove this method since its used once.

Reply:
when setState is moved into componentDidUpdate, linter throws error 'Do not use setState in componentDidUpdate'. How should I proceed?